src/services/get-staticQueries.js

```diff
@@ -0,0 +1,51 @@
+export default function getStaticQueries(string, queryElement, queryArrayA, queryArrayB) {
```

Comment:
Naming a variable just `string` is very unintuitive. Please consider using a more describing name, such as `key` or `subject`